packages/manager/modules/hub/src/components/payment-status-tile/AutorenewService.class.js

```diff
@@ -0,0 +1,63 @@
+import includes from 'lodash/includes';
+import snakeCase from 'lodash/snakeCase';
+
+import { DEBT_STATUS } from './payment-status-tile.constants';
+
+export default class AutorenewService {
+  constructor(service) {
+    Object.assign(this, service);
+
+    this.isExpired = this.status.toLowerCase() === 'expired';
+    this.isResiliated = this.getResiliated();
```

Comment:
I don't like the fact that you are providing the same information both statically and dynamically. It's either a dynamic value or a static one that never change.

If you want to avoid calling the method getResiliated multiple time you should use one time binding then